TeamCode/src/main/java/org/firstinspires/ftc/teamcode/StarterBot2024Teleop.java

```diff
@@ -173,7 +176,7 @@ public void loop() {
                 armRight.setPower(armPower);
                 armLeft.setMode(DcMotor.RunMode.RUN_TO_POSITION);
                 armRight.setMode(DcMotor.RunMode.RUN_TO_POSITION);
-                wrist.setPosition(wristUpPosition);
```

Comment:
since we are not using gamepad1 for arm or wrist control, delete all code related to it.

Reply:
ok